quickstart/301-aks-private-cluster/dns-zone-link.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Query AKS private DNS zone and then link it to custom DNS VNet.
```

Comment:
Why does this need to be done with a script? Can't you not use a combination of the azurerm_* and the dns_* providers to accomplish this?

Reply:
because the private dns zone name is unknown until we query it here.